jest/jest.ml

```diff
@@ -232,7 +234,10 @@ module Runner (A : Asserter) = struct
 
     let testAll name inputs callback =
       inputs |> List.iter (fun input ->
-        let name = {j|$name - $input|j} in
+        let name =
+          let input: string = Obj.magic input in
```

Comment:
Could be an identity external. Every time I see Obj.magic I remember the story with the syringes and the nice neighborhoods by Monsieur Leroy. 😄

Reply:
what's this story referring to?

Reply:
took a while to find it as the old maillist server is not working anymore: https://sympa.inria.fr/sympa/arc/caml-list/2005-07/msg00223.html